### DISCUSSION TITLE
Assigning upper and lower bounds to variables does not work

### DISCUSSION THREAD

**Discussion by BQKS** [yalmip/YALMIP#1457] [Problems using YALMIP] [answered]
Hello, teacher, when use Yalmap+gurobi 12.0+matlab, find that when assigning upper and lower bounds to variables, it not work because the mps format file displays the variables are free, so why, whether the gurobi 12 and yalmap are not compatible?

**Comment by johanlofberg** [comment, marked as answer]
YALMIP places all constraints in the list of general constraints, and gurobi will then extract trivial bounds from there
